CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ endif()
 FetchContent_Declare(
   GraphStreamingCC
   GIT_REPOSITORY      "https://github.com/GraphStreamingProject/GraphStreamingCC"
-  GIT_TAG             "cc_improve"
```

Comment:
what is CC_improve and how did it contribute to testing?

Reply:
So cc_improve was the branch where I multithreaded the connected components function. Currently, we're pointing at main since those changes were pulled in.